docs/source/logging/logging.md

```diff
@@ -57,25 +76,37 @@ For instance, you can enable the display of local variables inside `logging.yml`
 
 ```yaml
 rich:
-  class: kedro.extras.logging.RichHandler
+  class: kedro.logging.RichHandler
   rich_tracebacks: True
   tracebacks_show_locals: True
 ```
 
 A comprehensive list of available options can be found in the [RichHandler documentation](https://rich.readthedocs.io/en/stable/reference/logging.html#rich.logging.RichHandler).
 
+## Enable file-based logging
 
-### Use plain console logging
+File-based logging in Python projects aids troubleshooting and debugging. It offers better visibility into application's behavior and it's easy to search. However, it does not work well with read-only system such as [Databricks Repos](https://docs.databricks.com/repos/index.html).
+
+To enable file-based logging,  add `info_file_handler` in your `root` logger as follows in your `conf/logging.yml` as follow:
```

Comment:
Instructions out of date as `conf/logging.yml` will be removed

Reply:
It's not a must but we recommend user to keep their `loggging.yml` in `conf`. Is it confusing to reference it as `conf/logging.yml` or should I just reference it as `logging.yml`?

Reply:
I think putting this file in `conf` is natural, but we should make it clear that users need to create that file themselves and that it doesn't already exist.